models/ed/R/model2netcdf.ED2.R

```diff
@@ -760,7 +778,15 @@ put_T_values <- function(yr, nc_var, out, lat, lon, begins, ends, ...){
   out <- conversion(47, yr2s)  ## kg C m-2 yr-1 -> kg C m-2 s-1
   nc_var[[s+47]]<- ncdf4::ncvar_def("SoilResp", units = "kg C m-2 s-1", dim = list(lon, lat, t), missval = -999, 
                                     longname = "Soil Respiration")
-  
+  # SLZ specific hack until I figure that out - need to remove SLZ from out
+  if(!is.null(out$SLZ)){
+    out$SLZ <- NULL
```

Comment:
This is almost definitely fine, but I tend to prefer to use `out[["SLZ"]]` for stuff like this because `$` annoyingly does partial matching. So, if our object had `out$SLZ_except_not`, these calls would modify it. In practice, if we're creating `out` safely, it probably doesn't matter. Just food for thought.